es/13_1_Escribiendo_Puzzle_Scripts.md

```diff
@@ -0,0 +1,523 @@
+# 13.1: Escribir guiones de rompecabezas
```

Comment:
- `# 13.1: Escribiendo script rompecabezas`
- I'm not quite conform with the translation of puzzle as "rompecabezas", I was thinking in "acertijo" but that doesn't seem correct neither. What do you think?
- Possible future title: `Escribiendo scripts con acertijos`. Open to discussion. To change in #377 .

Reply:
I would prefer rompecabezas,  although acertijos is not bad.

Reply:
I mean, the idea of `puzzle` in the sentence is to imply something that must be completed, but in spanish the word `rompecabezas` is more related to game than to `missing piece` as the redeemScript should be in this cases.

I think that we should find a better word to express the idea above, but I approve this.

Reply:
We will receive feedback from Spanish community and maybe others words.